### ISSUE TITLE
New flex footer causing ellipsis overflow

### ISSUE THREAD

**Issue by baileyherbert** [squidfunk/mkdocs-material#3302]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

The "next" and "previous" links in the footer are ellipting with the new flex footer regardless of how short they are.

### Expected behaviour

"Classes"

![image](https://user-images.githubusercontent.com/9532377/144755980-c83a600b-c281-44f4-863d-0a766c0e3dca.png)

### Actual behaviour

"Class..."

![image](https://user-images.githubusercontent.com/9532377/144755953-f63f4d08-4922-4510-bac6-86b389dc2a2a.png)

### Steps to reproduce

Reproducible on a fresh setup with minimal config

### Package versions

- Python: `3.10.0`
- MkDocs: `1.2.3`
- Material: `8.0.4+insiders.4.3.0`

### Configuration

```yaml
theme:
  name: material
  features:
    - navigation.top
    - navigation.tracking
    - navigation.sections

site_name: Test

nav:
  - Introduction: index.md
  - Classes: classes.md
```

### System information

- Operating system: Win11
- Browser: Chrome (latest stable)

**Comment by squidfunk** [comment]
Thanks for reporting. I'm not able to reproduce this in Chrome, macOS.

**Comment by baileyherbert** [comment]
Thanks @squidfunk, can confirm that it's working normally in Firefox from here, it's just Chrome/Edge for me and I can see it on your official docs too. Boy do I hate flexboxes! 🤣 

Cheers!

**Comment by squidfunk** [comment]
Potential fix in 37e6f430f.

**Comment by baileyherbert** [comment]
Yup, it's working perfectly now! Thank you! ❤️

**Comment by squidfunk** [comment]
Let's leave this open until it's released 😊

**Comment by squidfunk** [comment]
We can in fact close this since it happened after the 8.0.4. release, so it's not a regression as the changes inducing the bug reported were not released in the first place. Thus, no bugfix release to be made 😉

**Comment by squidfunk** [comment]
Released as part of 8.0.5.
